src/tests/stress/pedsim/README.md

```diff
@@ -0,0 +1,5 @@
+This folder contains the implementation of a pedestrian simulation in Encore. The simulation uses input XML files, which by conventions have the name X_scenario.xml, where X is the number of agents they contain. The various test instances uses different files split into different numbers of regions.
+
+Note: The number of parallel regions/active-object workers is the input number squared. So regions(agents,3) will make 9 workers.
+
```

Comment:
The description of the PedSim could be briefly summarised here, IMO. Or the link the the document.